demo/screens/ListSellOrdersScreen.js

```diff
@@ -49,32 +50,42 @@ export default class ListLandsScreen extends React.Component {
     const hasOrder = parseInt(orderId, 16) !== 0;
     if (!hasOrder) throw Error(`Estate (id:${estateId}) has no sell order.`);
 
-    const priceMana = Number(price.toString()) / 1e18;
+    // Note: Conversion to USD will be implemented on v0.2.0
     const manaPerUsd = 30;
+    const priceMana = Number(price.toString()) / 1e18;
     const priceUsd = Number(priceMana / manaPerUsd).toFixed(2);
     const imgUrl = `https://api.decentraland.org/v1/estates/${estateId}/map.png`;
 
     return {
-      id: estateId,
-      name: estateName,
+      id: orderId,
       priceMana,
       priceUsd,
-      img: imgUrl,
-      orderId,
       seller,
       expiresAt,
+      asset: {
+        id: estateId,
+        name: estateName,
+        img: imgUrl,
+      },
     };
   }
 
-  renderRow = row => {
-    const { item: land } = row;
+  renderItem = ({ item: sellOrder }) => {
     const handlePress = () =>
-      this.props.navigation.navigate("LandClaimScreen", { land });
-
-    return <LandRow id={land.id} land={land} onPress={handlePress} />;
+      this.props.navigation.navigate("SellOrderScreen", { sellOrder });
+    return (
+      <SellOrdersListItem
+        id={sellOrder.id}
```

Comment:
Why pass in id inside the `sellOrder` object and separately?

Reply:
I've seen that way on the `FlatList` doc and I've thought that was mandatory but it isn't. I've just removed that.

Reply:
Done.

Reply:
Thanks! Resolving.